examples/l1-resources/stream-connection-cluster.ts

```diff
@@ -1,32 +1,31 @@
 import * as cdk from 'aws-cdk-lib';
 import { Construct } from 'constructs';
-import { CfnStreamConnection, CfnStreamConnectionPropsType } from 'awscdk-resources-mongodbatlas'
-import { env } from 'node:process';
+import { CfnStreamConnection, CfnStreamConnectionPropsType, DbRoleToExecute, DbRoleToExecuteType } from 'awscdk-resources-mongodbatlas'
 
 interface AtlasStackProps {
 	readonly projectId: string;
 	readonly profile: string;
 	readonly instanceName: string;
 	readonly connectionName: string;
-	readonly type: CfnStreamConnectionPropsType;
 	readonly clusterName: string;
 }
 
 const app = new cdk.App();
 
 
-export class CdkTestStack extends cdk.Stack {
+export class CdkTestStackCluster extends cdk.Stack {
 	constructor(scope: Construct, id: string, props?: cdk.StackProps) {
 		super(scope, id, props);
 
 		const atlasProps = this.getContextProps();
-		const streamConnection = new CfnStreamConnection(this, "stream-connection-testing-stack", {
+		const streamConnection = new CfnStreamConnection(this, "stream-connection-cluster-stack", {
 			profile: atlasProps.profile,
 			instanceName: atlasProps.instanceName,
 			projectId: atlasProps.projectId,
 			connectionName: atlasProps.connectionName,
 			type: CfnStreamConnectionPropsType.CLUSTER,
-			clusterName: atlasProps.clusterName
+			clusterName: atlasProps.clusterName,
+			dbRoleToExecute: {role: "atlasAdmin", type: DbRoleToExecuteType.BUILT_UNDERSCORE_IN}
```

Comment:
okay `BUILT_UNDERSCORE_IN` is soo weird lol
I know it's unrelated but could we rename this? I know it's a breaking change but I think now might be the right time before users actively start using this 
Or let's at least create a ticket before this GA's
@Zuhairahmed WDYT?

Reply:
This commit dfb3bfe, adds a step to the `cdk.sh` script.
It calls a python script that removes `UNDERSCORE_` from names with `{prefix}_UNDERSCORE_{suffix}`.
After researching `projen` I think this is the best approach:

1. The `cdk.sh` seems to only be called locally and no changes to the ci workflows are needed
2. Using the `cdk.sh` we modify the variable name in the earliest moment and avoid having to do any changes to the `projen` magic

The alternative of using `projen` and adding modificaitons to `.projenrc.js` seems possible, but I think it is an advantage to keep the names generated in the `src/*` unchanged so the names matches the repo state

Reply:
This is great @EspenAlbert ! Thanks for looking into it:)
any chance you had a look if we have other enums with `_UNDERSCORE_` as well? or is this resource the only one?

Reply:
ok I had a look and this is the only one so we are good there

Reply:
Thanks.  I included this change to this branch now 👍

Reply:
@EspenAlbert Confirmed with @Zuhairahmed , currently no users for this CFN resource so we should go ahead and release this change